src/main/java/com/github/minecraftschurlimods/arsmagicalegacy/common/spell/shape/Beam.java

```diff
@@ -37,4 +37,8 @@ public boolean needsPrecedingShape() {
     public boolean needsToComeFirst() {
         return true;
     }
+
+    public static int getBeamColor(ISpell spell, List<ISpellModifier> modifiers) {
+        return spell.primaryAffinity().color(); //TODO color modifier
+    }
```

Comment:
why is this here?

Reply:
leftover from before the color modifier was merged